### PR TITLE
Benchmarks: Build Pipeline - Add perftest as a submodule and add build logic

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Add perftest as a submodule of superbench.

**Major Revision**
- Add perftest as submodule
- Add build script for it.

### CODE REVIEW

File: third_party/Makefile [hunk 1]
```diff
@@ -29,3 +29,9 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+# Build perftest.
+# The version we use is v4.5-0.2, which is the latest release tag of perftest
+perftest: 
+ifneq (,$(wildcard perftest/autogen.sh))
+	cd perftest && ./autogen.sh && ./configure CUDA_H_PATH=/usr/local/cuda/include/cuda.h --prefix=$(SB_MICRO_PATH) && make -j && make install
+endif
```
Comment (cp5555) [inline third_party/Makefile:37]: Please add new line.

**Review by cp5555 — APPROVED** [review]

File: third_party/Makefile [hunk 2]
```diff
@@ -29,3 +29,9 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+# Build perftest.
```
Comment (abuccts) [inline third_party/Makefile:32]: add an empty line between different targets?

File: third_party/Makefile [hunk 3]
```diff
@@ -29,3 +29,9 @@ bandwidthTest: sb_micro_path
...
+# The version we use is v4.5-0.2, which is the latest release tag of perftest
```
Comment (abuccts) [inline third_party/Makefile:33]: it's confusing, the latest release tag changes over time

**Review by abuccts — COMMENTED** [review]
pls remove it in dockerfile

**Review by guoshzhao — APPROVED** [review]

File: third_party/Makefile (hunk 2 above)
Comment (yukirora) [reply third_party/Makefile:45]: ok, changed

File: third_party/Makefile (hunk 3 above)
Comment (yukirora) [reply third_party/Makefile]: delete the the latter part

File: third_party/Makefile (hunk 1 above)
Comment (yukirora) [reply third_party/Makefile]: ok

**Review by abuccts — APPROVED** [review]

**Pull request merged**